src/Microsoft.VisualStudio.Services.Agent/ProcessInvoker.cs

```diff
@@ -241,4 +315,14 @@ public ProcessExitCodeException(int exitCode, string fileName, string arguments)
             ExitCode = exitCode;
         }
     }
+
+    public class ProcessDataReceivedEventArgs : EventArgs
```

Comment:
Is this class needed? It looks like it has the same interface as DataReceivedEventArgs with the exception of the constructor.